pkg/synopsysctl/cmd_create.go

```diff
@@ -920,10 +920,10 @@ var createPolarisReportingCmd = &cobra.Command{
 			}
 		}
 
-		// Check Helm Command with --dry-run before deploying any resources
-		out, err := util.RunHelm3("install", []string{polarisReportingName, polarisReportingChartRepository, "-n", namespace, "--dry-run"}, helmValuesMap)
+		// Check Dry Run before deploying any resources
+		err = util.CreateWithHelm3(polarisReportingName, namespace, polarisReportingChartRepository, helmValuesMap, kubeConfigPath, true)
 		if err != nil {
-			return fmt.Errorf("failed to create Polaris-Reporting resources: %+v; %+v", out, err)
+			return fmt.Errorf("failed to create Polaris-Reporting resources: %+v", err)
```

Comment:
This is a nice idea!  Can we indicate in the error we failed during a dry-run?

Reply:
Actually, I don't think this dry-run is needed, can we remove this?

Reply:
It's not needed, but it's nice to do a dry run before we deploy anything into the cluster (the secrets that we make that are not in the helm chart)